src/main/resources/db/changelog/changes/v2.1.0/changelog-v2.1.0.xml

```diff
@@ -27,4 +27,26 @@
     </createView>
   </changeSet>
 
+  <changeSet id="create_view_ledger" author="kjain@ebsco.com">
+    <preConditions onFail="CONTINUE">
+      <tableExists tableName = "ledger" schemaName="${tenant_id}_mod_finance_storage"/>
+    </preConditions>
+    <createView
+      replaceIfExists="true"
+      viewName="src_ledger">
```

Comment:
nit: it might be good to prepend these views with the module they come from like we do with a lot of the other ones. So like src_finance_ledger